Assets/MixedRealityToolkit.Tests/PlayModeTests/BoundingBoxTests.cs

```diff
@@ -152,7 +152,7 @@ public IEnumerator ScaleMinMax()
             float maxScale = 2f;
 
             var bbox = InstantiateSceneAndDefaultBbox();
-            var scaleHandler = bbox.EnsureComponent<TransformScaleHandler>();
+            var scaleHandler = bbox.EnsureComponent<MinMaxScaleConstraint>();
```

Comment:
Breaking change, or does the migrationhandler help with this?

Reply:
that is fine - it's just a rename - we don't need a migration for that - unity is smart enough to figure that out

Reply:
I wonder if we can use `[MovedFrom(true, sourceClassName:"TransformScaleHandler")]` to ease migration in code

Reply:
The `TransformScaleHandlers` in HandInteractionExamples have been updated to `MinMaxScaleConstraints` without any changes to that scene in this PR.

Reply:
We tried looking into the mentioned attribute MovedFrom but couldn't find any documentation about it. What does this attribute do? 
Unity handles renames automatically - there's no need to do any migration. All references to renamed scripts are still working. I'm not sure I'm understanding the concern about the rename.